basic_autonomy/include/basic_autonomy/basic_autonomy.hpp

```diff
@@ -34,6 +34,7 @@
 #include <unordered_set>
 #include <autoware_msgs/msg/lane.h>
 #include <lanelet2_core/geometry/Point.h>
+#include <basic_autonomy/impl/basic_autonomy.tpp>
```

Comment:
Not a comment on this specific line but for the PR description. Could you update the text for "have many duplicate points. See below picture" to explain that the image/map has been modified to show all of the points that were originally overlapping (at least I think this is what is being shown)?

Additionally, could we consider updating the PR name to reflect that this is a fix for filtering/removing duplicate lane centerline points during trajectory generation?

Reply:
Added these changes